### PR TITLE
repository_class configuration option could be replaced with FQCN repository name

### PR DESCRIPTION
**Pull request by kejwmen**
Implements #5.

### CODE REVIEW

File: src/DependencyInjection/ProophEventStoreExtension.php [hunk 1]
```diff
@@ -172,6 +173,15 @@ private function loadEventStore(string $name, array $options, ContainerBuilder $
 
         if (! empty($options['repositories'])) {
             foreach ($options['repositories'] as $repositoryName => $repositoryConfig) {
+                $repositoryClass = $repositoryConfig['repository_class'] ?? $repositoryName;
+
+                if (! class_exists($repositoryClass)) {
+                    throw new RuntimeException(\sprintf(
```
Comment (UFOMelkor) [inline src/DependencyInjection/ProophEventStoreExtension.php]: Please remove the leading `\` for consistency

**Comment by UFOMelkor** [comment]
Docs are already added in #27

File: src/DependencyInjection/ProophEventStoreExtension.php (hunk 1 above)
Comment (kejwmen) [reply src/DependencyInjection/ProophEventStoreExtension.php]: :ok_hand:

**Comment by kejwmen** [comment]
done, thanks for taking care of docs.

File: src/DependencyInjection/ProophEventStoreExtension.php [hunk 2]
```diff
@@ -176,7 +176,7 @@ private function loadEventStore(string $name, array $options, ContainerBuilder $
                 $repositoryClass = $repositoryConfig['repository_class'] ?? $repositoryName;
 
                 if (! class_exists($repositoryClass)) {
-                    throw new RuntimeException(\sprintf(
+                    throw new RuntimeException(sprintf(
```
Comment (sandrokeil) [inline src/DependencyInjection/ProophEventStoreExtension.php]: Why was the `\` backslash removed from `sprintf`? This gives a minimal [performance gain](https://github.com/Roave/FunctionFQNReplacer#benchmark).

Comment (prolic) [reply src/DependencyInjection/ProophEventStoreExtension.php]: I've read somewhere, that the performance difference will be gone in future php version and you shouldn't use this except when needed (but I can't find the sources for this again at the moment). Currently we don't do this within all the prooph components, so even if my statement above would be invalid, let's keep it consistent.

Comment (kejwmen) [reply src/DependencyInjection/ProophEventStoreExtension.php]: and tools like cs-fixer can always change it in entire project

**Comment by UFOMelkor** [comment]
Thank you @kejwmen

**Pull request merged**